Assets/Scripts/Managers/AdultManager.cs

```diff
@@ -0,0 +1,52 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+public class AdultManager : SingletonBehaviour<AdultManager>
+{
+	[SerializeField] GameObject _adultPrefab = null;
+
+	[Range( 0.0f, 1.0f )]
+	[SerializeField] float _goodBadCutoff = 0.5f;
+
+	private List<AdultSpawnTag> _spawnPoints = new List<AdultSpawnTag>();
+	private List<ResourceSpawner> _resourceSpawners = new List<ResourceSpawner>();
```

Comment:
Actually, consider having both of these be lists of transforms and naming them good/badBuddySpawnPoints and then use LINQ or something to get the transform in your FindObjectsOfType<X> below.